src/components/pages/docs/Sap.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from "react";
+
+const Sap: FC = () => {
+ return (
+  <article className="prose prose-stone m-auto text-justify max-w-3xl">
+   <h1 className="font-tall uppercase text-leaf">ToS Definitions</h1>
+   <h2 className="uppercase font-tall text-root">Roots</h2>
```

Comment:
Wrap each of the `h2` and `p` on a section with an ID, add a table of content with links to each section below the `h1`.